lib/experimental-default-theme.json

```diff
@@ -169,10 +169,10 @@
 			"spacing": {
 				"customPadding": false,
 				"units": [ "px", "em", "rem", "vh", "vw" ]
-			},
-			"border": {
-				"customRadius": true
-			}
+			},	
+		},
+		"border": {
+			"customRadius": true
```

Comment:
I guess both border  and  spacing seem misplaced right? @nosolosw @jorgefilipecosta

Reply:
The hierarchy seems fine to me in `master` (`color`, `typography`, `spacing`, and `border` as subkeys of `settings`):

![Captura de ecrã de 2020-12-29 10-57-26](https://user-images.githubusercontent.com/583546/103275524-ae341a80-49c4-11eb-84f0-e5916ff12523.png)